ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1993,7 +1993,8 @@ ntp:
   authentication_keys:
     - id: < key_identifier | 1-65534 >
       hash_algorithm: < md5 | sha1 >
-      key: "< type7_obfuscated_key >"
+      key: "< obfuscated_key >"
+      key_type: < 0 | 7 | 8a | default -> 7 >
```

Comment:
The default 7 will be implemented in 4.0, so let's remove it from the current documentation.
```suggestion
      key_type: < 0 | 7 | 8a >
```

Reply:
Maybe this is the right place to add a description
`# From AVD4.0 the value of key_type will be 7 by default`